docs/Developer-Tips.md

```diff
@@ -47,3 +47,112 @@ Then to regenerate the Java code, run
 ```bash
 $ bin/tachyon protoGen
 ```
+
+### Full list of the commands in bin/tachyon
+
+Most commands in `bin/tachyon` are for developers. The following table explains the meaning and the
+syntax of each command.
+
+<table class="table table-striped">
+<tr><th>Command</th><th>Args</th><th>Meaning</th></tr>
+<tr>
+  <td>format</td>
+  <td>[-s]</td>
+  <td>Format Tachyon Master and all Workers. The option [-s] indicates that only format when underfs
```

Comment:
NIT: The option [-s] indicates that the command should only format when the underfs is local and doesn't already exist.